### ISSUE TITLE
Export in svg/pdf

### ISSUE THREAD

**Issue by francicco** [GeneMANIA/genemania#174]
Hi,
 
I’m using your GeneMANIA server, for some analyses. I see that I can only save the network as jpg, would it be possible to export it in a vectorial format?
 
Thanks a lot
Francesco

**Comment by risserlin** [comment]
I don't think you can export it to different formats on the web version but you can from the Cytoscape version if you have Cytoscape installed.

**Comment by francicco** [comment]
Hi @risserlin,

Yes, I tried, but I didn't really like the layout. There's no way you can update the server? : )

Cheers
F

**Comment by risserlin** [comment]
There are many different layouts that you can try from Cytoscape.
![Screen Shot 2022-02-17 at 1 22 12 PM](https://user-images.githubusercontent.com/3842763/154545970-0abc1562-4c23-463b-8e52-f0e7236c029d.png)

**Comment by francicco** [comment]
I knew that...
Anyways, thanks a lot
F
